components/overflow-group/overflow-group-mixin.js

```diff
@@ -318,18 +328,28 @@ export const OverflowGroupMixin = superclass => class extends LocalizeCoreElemen
 
 		this._updateOverflowItemsRequested = true;
 		setTimeout(() => {
-			this._overflowItems = this._slotItems.map(node => this.convertToOverflowItem(node));
+			this._itemLayouts = this._getItemLayouts(this._slotItems);
+			this._overflowItems = this._slotItems.map((node) => this.convertToOverflowItem(node));
+			this._chomp();
 			this._updateOverflowItemsRequested = false;
 			this.requestUpdate();
 		}, 0);
 	}
 
-	_handleResize(entries) {
+	async _handleResize(entries) {
+		await (document.fonts ? document.fonts.ready : Promise.resolve());
```

Comment:
I found that if we didn't wait for this then the item widths calculated would usually not be correct. It wouldn't be crazy off, but for test cases that were exact it was noticeable and caused flake.

I put this here as I wanted to wait for it before the resize observer got used. It could also have a bool where we only await it if they haven't yet loaded.

Reply:
Makes sense to me, but maybe a comment here would be helpful so we remember why we did this